Source/FullScreenPokemon.ts

```diff
@@ -4888,6 +4888,52 @@ module FullScreenPokemon {
             player.FSP.killNormal(player.bordering[player.direction]);
         }
 
+        /**
+         * Makes a StrengthBoulder move.
+         *
+         * @param player   The Player.
+         * @param pokemon   The Pokemon using Strength.
+         * @todo Eventually add check to make sure the Player beat the Gym leader needed to use the move.
+         * @todo Add an animation for what happens when the StrengthBoulder is moved.
```

Comment:
This looks like it is the animation :smile: can you change the todo to say the exact speed and sound should be verified?